src/app/app.js

```diff
@@ -2,7 +2,7 @@ import React from "react";
 import { Router, Link } from "@reach/router";
 
 import { CreateChallenge } from "../views/views";
-import { Chrome, ChallengeWidget } from "../components/components";
+import { Chrome, ChallengeWidget, Label } from "../components/components";
```

Comment:
I don't think you need to showcase the `Label` component in this file directly.
You can use it inside the `ChallengeWidget` component instead.

Reply:
if I remove ´Label´ from this component, the app will not compile, it says Label is not defined